Makefile

```diff
@@ -1,9 +1,6 @@
-KUBE_SCHEMA_ORG:=jkcfg
-KUBE_SCHEMA_REPO:=kubernetes-schema
-KUBE_SCHEMA_SHA1:=83d92a798500efd744a576df994196e75f3133dd
-SCHEMA_ZIP:=build/${KUBE_SCHEMA_ORG}-${KUBE_SCHEMA_REPO}-${KUBE_SCHEMA_SHA1}.zip
-# the next one is a consequence of how the zip file is made: the files
-# will be in a directory named for the repo and commit hash.
+KUBE_SCHEMA_ORG:=yannh
+KUBE_SCHEMA_REPO:=kubernetes-json-schema
+KUBE_SCHEMA_SHA1:=master
```

Comment:
What I was trying to achieve with using a SHA1 here was that it would rebuild the schemas directory when the targeted commit changed, but not otherwise.

Changing it to a branch name means it will run once and never again (without `make clean`). Any automated builds will pick up whatever is in master at the time, so won't be repeatable.

Reply:
ah yes. that makes sense. changing to the recent HEAD commit.